### DISCUSSION TITLE
Overclock option fix ( I think I have an idea!) RG280V

### DISCUSSION THREAD

**Discussion by EnriqueVill** [eduardofilo/RG350_adam_image#97] [General]
I don't think Overclocking corrupts the SD. It seems like the device remembers how much power to draw extra for overclocking even after shutdown resulting in a dim lit LED Indicator when the device is off. I'm guessing the fix will be to add:
Set "Overclocking: NO" When:
-exited rom
-shutdown selected
-On bootup
I think that idea could help.
You could also add a script application to manually reset Overclock settings to prevent needing to re-flash the OS.
Great Firmware, it's freaking amazing! Thanks!

P.S. - Sorry I sent an Email, I had no idea I could just post here haha

**Comment by eduardofilo** [comment]
Interesting theory. But I have no possibility to test and therefore develop that script. I have also not experienced a card corruption yet. Although neither of my two consoles support OC not even 1092.
People say that when the console does not boot after such an episode, the partition is seen corrupted with a partition editor on the PC, so I interpret that the problem is real. And it does seem feasible to me that it happens when applying the OC, since the processor goes crazy and the only part of the console that can be modified is the SD.

**Comment by EnriqueVill** [comment]
By overclocking the CPU or GPU, you're configuring the System BIOS which is part of the motherboard, not the OS. So you would be modifying both the device and the Sdcard, no?
 _(People might have found corruption from Overclocking because their game froze and were forced to **Hard reset** the device or by pulling out the SD causing corruption.)_
On my end, SDcard(1) Both partitions didn't show corrupt at all; they were healthy according to the windows partition manager. The Roms SDcard(2) was the only one windows asked to fix and luckily found no bad sectors.
 I noticed The LED Power indicator on the device was still on yet, **dimly lit** even without both SDcards in the device; Which signifies some kind of memory being held within the motherboard (most likely caused by the Overclocking since that is really the only feature that reconfigures the System BIOS.)
 
After a fresh reflash, I played with the RG280V all day and before **shutdown**, I make sure to **"select"** >**"Overclocking: No"** for my roms and It's been working perfectly. You could also turn off Overclocking by playing a different game with default to **"overclocking:No"**  (those who don't run into issues most-likely don't touch Overclock.)

other than that, this is really well ported software! Thanks for the support! love this Image.
If you intend to create a fix for OC and need me to test it on my RG280V, I'd be happy to help give you feedback on it:
Vega.contacts@gmail.com

Overclock Performance?:
OC: OFF - Street Fighter III runs half speed
OC: ON - Street Fighter III runs full speed

Edit: I would offer to take a look at the code myself but I don't even know how to code but I think I understand the fundamentals. Hopefully I've been helpful somehow lol

**Comment by eduardofilo** [comment]
Thanks for your suggestions, but I can't really figure out how to fit these ideas together. Because as I interpret it there are two possibilities:

1. The processor supports OC => No problem. Also, when you exits game, SimpleMenu returns processor to normal speed.
2. The processor doesn't support OC => There is no opportunity to disable the OC option with any script.

I also believe that if not all, most of the corruptions are due to pressing the reset. Some user has told me that the corruption occurred after using the suspend option several times. But it is very difficult to draw conclusions from statements like this. As long as someone doesn't find a way to reproduce the problem systematically, I don't think much can be done.

In any case I suppose that the solution will be more in the hands of the developers of OD Beta or even SimpleMenu. I'm going to close the issue because I don't see how I can find or provide the solution. Thanks again for your contributions.

**Comment by EnriqueVill** [comment]
OH! I'm sorry, I assumed you had total control over the Simple Menu lol Thank you for your support and porting it over brother, means alot!
